index.html

```diff
@@ -0,0 +1,30 @@
+<!--HTML layout page-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>Portfolio | Sully</title>
+</head>
+<body class="main-content">
+    <header class="mainheader">
+        <h2 class="usernamehandle">bettercallsully</h2>
+        <nav class="toolbar">
+            <img class="hamburger" src="myassets/hamburger.png" alt="Hamburger Logo">
+        </nav>
+    </header>
+    <article>
+         <section class="articletext">
+             <h2 class="introduction">I'm Sully<br>Glad to see you!</h2>

```

Comment:
- [ ] Kindly notice that this element is the main title (heading) in the website so, it'd be better if it has a `<h1>` instead.
It helps with the [SEO](https://moz.com/beginners-guide-to-seo) for the website.